runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/AutocompleteWidget.tsx

```diff
@@ -45,7 +46,8 @@ const AutocompleteWidget: FC<WidgetProps & { groupBy?: (option: any) => string }
                     label={label}
                     onChange={(event) => onChange(event, event.target.value)}
                     InputLabelProps={{ shrink: true }}
-                    error={!!props.rawErrors}
+                    error={!!customErrors || !!rawErrors}
```

Comment:
Use `Boolean(foo)` its much more readable

Reply:
yup, try to avoid JS specific operations like `!!` or plus before string to parse it to the number

Reply:
roger that